app/controllers/pages_controller.rb

```diff
@@ -1,25 +1,20 @@
 class PagesController < ApplicationController
-  def home
-  end
-
-  def about
-  end
-
-  def resources
+  def adopt
```

Comment:
So I'm going to jump in here and hopefully this won't come across too harshly, but I feel rather strongly about this so here it goes. When designing a controller, a best practice is to limit the names of the actions (public methods) that a controller has to one of the following seven: index, show, new, create, edit, update, and destroy. The reason for this is to help convey a contract between developers about what to expect in a controller. At the moment your PagesController is on it's way to becoming cluttered with lots of different logic in your application. I would strongly suggest moving this action into a more adequately named controller in order to reflect this. What would you think about having this as an `index` action in a `RatsController`?